packages/web/src/scss/components/Footer/README.md

```diff
@@ -0,0 +1,239 @@
+- Use [Grid][grid] and [Flex][flex] components to create the desired layout.
+- Use the [Divider][divider] component to separate individual sections.
+- Use spacing utility classes like `mb-*` or `pt-*` to achieve desired spacings between components.
+
+## Navigation Links
+
+Navigation links are structured in sections with a headline and a [Stack][stack] of links.
+
+👉 Please note how the `<nav>` element is paired with an `<h3>` element (using the `aria-labelledby` attribute) to
+provide a semantic connection between the headline and the navigation component.
+
+```html
+<nav aria-labelledby="footer-part-one">
+  <h3 class="typography-body-large-text-bold mb-600" id="footer-part-one">Section headline</h3>
+  <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+    <li>
+      <a href="https://www.example.com">Link</a>
+    </li>
+    <li>
+      <a href="https://www.example.com">Link</a>
+    </li>
+    <li>
+      <a href="https://www.example.com">Link</a>
+    </li>
+  </ul>
+</nav>
+```
+
+You can use as many navigation sections like this as you need and wrap them in a responsive [Grid][grid] layout.
+
+## Product Logo, Social Media Links and Language Switch
+
+This section is optional and consists of a [Grid][grid] layout with up to three (also optional) columns:
+
+```html
+<div
+  class="Grid Grid--cols-1 Grid--desktop--cols-3 Grid--alignmentXCenter Grid--desktop--alignmentXStretch Grid--alignmentYCenter"
+  style="--grid-spacing-x: var(--spirit-space-900); --grid-spacing-y: var(--spirit-space-900)"
+>
+  <div class="text-desktop-left">
+    <!-- Product logo -->
+  </div>
+  <!-- Flex with social media links -->
+  <div class="text-desktop-right">
+    <!-- Language switch -->
+  </div>
+</div>
+```
+
+👉 Please mind the `text-desktop-left` and `text-desktop-right` utility classes. They are used to align individual
+grid columns to the left and right side of the container on desktop screens.
+
+### Product Logo
+
+Use the [Product Logo][product-logo] component to display the logo of your product.
+
+### Social Media Links
+
+Use the secondary [Button][button] component to create social media links inside a [Flex][flex] container.
+
+```html
+<div class="Flex Flex--row Flex--wrap Flex--alignmentXCenter Flex--alignmentYCenter">
+  <!-- Repeat the `<a>` block for each social media link. -->
+  <a href="https://www.example.com" class="Button Button--secondary Button--medium Button--square">
+    <span class="accessibility-hidden">Facebook</span>
+    <svg width="24" height="24" aria-hidden="true">
+      <use xlink:href="/assets/icons/svg/sprite.svg#logo-facebook" />
+    </svg>
+  </a>
+</div>
+```
+
+### Language Switch
+
+Use the [Select][select] component to create a language switch, or list the languages as country flags in a [Flex][flex]
+layout.
+
+## Secondary Links
+
+This section is optional and consists of a [Flex][flex] layout with secondary links.
+
+```html
+<ul
```

Comment:
shouldn't there be a <nav> element?

Reply:
I think something is missing in your message…

Reply:
Updated. Sorry, I skipped backticks :) 

the point is that this is basically a navigation, isn't it? So maybe it should be wrapped inside `nav`